identity-service/src/authMiddleware.js

```diff
@@ -56,11 +56,12 @@ async function authMiddleware (req, res, next) {
     let walletAddress = recoverPersonalSignature({ data: encodedDataMessage, sig: signature })
     const user = await models.User.findOne({
       where: { walletAddress },
-      attributes: ['id', 'blockchainUserId', 'walletAddress', 'createdAt']
+      attributes: ['id', 'blockchainUserId', 'walletAddress', 'createdAt'],
+      raw: true
```

Comment:
how come? (maybe comment)

Reply:
this is based on half-an-understanding as of now. one of those things I'm definitely going back to look at. Don't want to introduce any change here ideally and actually don't think I need to.